src/components/gcds-radio/gcds-radio.tsx

```diff
@@ -0,0 +1,127 @@
+import { Component, Element, Event, EventEmitter, Prop, Listen, Host, h } from '@stencil/core';
+import { assignLanguage } from '../../utils/utils';
+
+@Component({
+  tag: 'gcds-radio',
+  styleUrl: 'gcds-radio.css',
+  shadow: false,
+  scoped: true,
+})
+export class GcdsRadio {
+
+  @Element() el: HTMLElement;
+
+  private lang: string;
+  private shadowElement?: HTMLInputElement;
+
+  /**
+   * Id attribute for an input element.
+   */
+  @Prop({ reflect: true, mutable: true }) radioId!: string;
+
+  /**
+   * Form field label
+   */
+  @Prop({ reflect: true, mutable: false }) label!: string;
+
+  /**
+   * Name attribute for an input element.
+   */
+  @Prop({ reflect: true, mutable: false }) name!: string;
```

Comment:
I don't think we need a separate name prop, the name value is supposed to be the same as the ID so we can set it to the same as the ID prop.

Reply:
To group radio buttons together you assign them the same name. If we tie that to the ID prop, the labels would be broken and only select the first radio button that appears.